mentions/src/test/kotlin/io/spine/examples/pingh/mentions/given/PredefinedGitHubResponses.kt

```diff
@@ -91,4 +109,22 @@ public class PredefinedGitHubResponses : GitHubClientService {
     public fun unfreeze() {
         isFrozen = false
     }
+
+    /**
+     * Sets the HTTP status of the response.
+     *
+     * If the status code is different from 200, an exception will be thrown.
+     * After changing the status, it is necessary to return it to the default state,
```

Comment:
Not sure why this is necessary. It's a responsibility of whoever uses this API, and l suggest let them decide.